src/main/java/com/denizenscript/ddiscordbot/commands/DiscordCommand.java

```diff
@@ -92,117 +98,43 @@ public DiscordCommand() {
 
     public enum DiscordInstruction { CONNECT, DISCONNECT, MESSAGE, ADD_ROLE, REMOVE_ROLE, STATUS, RENAME, START_TYPING, STOP_TYPING, EDIT_MESSAGE, DELETE_MESSAGE }
 
-    @Override
-    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
-        for (Argument arg : scriptEntry) {
-            if (!scriptEntry.hasObject("id")
-                    && arg.matchesPrefix("id")) {
-                scriptEntry.addObject("id", new ElementTag(CoreUtilities.toLowerCase(arg.getValue())));

```

Comment:
old code force-lowercased `id` in parseArgs - that needs to be matched in the new autoExecute